detectron2/modeling/roi_heads/mask_head.py

```diff
@@ -27,8 +27,15 @@
 The registered object will be called with `obj(cfg, input_shape)`.
 """
 
+# To get torchscript support, we make mask head class `MaskRCNNConvUpsampleHead` a subclass
+# of `nn.Sequential`. If you want to add new layers in this head class, please make sure
+# they are added to the class in the order of that in the network.
+# todo: revert to nn.Module when torchscript supports List[nn.Module]
+# https://github.com/pytorch/pytorch/pull/38059
```

Comment:
Let's link to the issue instead. It's not certain yet whether the issue will be addressed by this PR or others

Reply:
In fact, after a second thought the changes in this class are still quite straightforward and don't have to be reverted. So I would suggest just removing these two lines of comments.

Reply:
for `mask_head`, `nn.Sequential` is indeed more straightforward since the network structure is 'sequential'. I deleted these two lines of comments.